table/internal_iterator.h

```diff
@@ -52,6 +52,7 @@ class InternalIteratorBase : public Cleanable {
   // All Seek*() methods clear any error status() that the iterator had prior to
   // the call; after the seek, status() indicates only the error (if any) that
   // happened during the seek, not any past errors.
+  // 'target' contains user timestamp if timestamp is enabled.
```

Comment:
Make sure Get() follows this rule when Seek() is called in BlockBasedTable::Get().

Reply:
Current and original assumption is that the `const Slice& key` argument given to `BlockBasedTable::Get()` is internal key, which contains timestamp if timestamp is enabled. Do you mean we should assert the `key.size()` must be greater than or equal to `8 + timestamp_size_`?

Reply:
I don't mean assertion. I just double check that whether it is consistent to how we already use internal iterator.